### ISSUE TITLE
Bangla font rendering problem in MathJax

### ISSUE THREAD

**Issue by irkreja** [mathjax/MathJax#2426]
Bangla Font got broken. Please fix this issue.

![IMG_20200214_225759](https://user-images.githubusercontent.com/29326759/74551527-8939e300-4f7d-11ea-92d6-cfca1f2cb536.jpg)

**Comment by dpvc** [comment]
You need to configure MathJax to use the surrounding font for text elements, and then put your Bangla text inside `\text{}` macros (or `<mtext>` elements if you are using MathML input).  E.g.

``` html
<script>
MathJax = {
  chtml: {
    mtextInheritFont: true
  }
}
</script>
<script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"></script>
```

with the math entered as, e.g.,

```
$$ \text{বিকৃতি} = ...$$
```

**Comment by jayprakash1** [comment]
I faced a similar issue with Hindi Font and made the following changes to fix it for Mathjax 2.7.5

```
<script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
<script>
window.addEventListener('load', function () {
  MathJax.Hub.Config({
    CommonHTML: {
      mtextFontInherit: true
    }
  });
});
</script>
```
